sw/device/tests/crypto/ecdsa_p256_functest.c

```diff
@@ -45,42 +46,25 @@ static void compute_digest(void) {
   memcpy(digest.h, hmac_digest.digest, sizeof(hmac_digest.digest));
 }
 
-bool sign_then_verify_test(void) {
+status_t sign_then_verify_test(void) {
   ecdsa_p256_signature_t signature;
   hardened_bool_t verificationResult;
 
   // Spin until OTBN is idle.
-  if (otbn_busy_wait_for_done() != kOtbnErrorOk) {
-    return false;
-  }
+  TRY(otbn_busy_wait_for_done());
 
   // Generate a signature for the message
```

Comment:
This PR is changing this line, but would you mind adding a period ('.') in this comment and the in the follow comments for consistency?